### DISCUSSION TITLE
Set remote schema from env variable

### DISCUSSION THREAD

**Discussion by henriktorget** [hasura/graphql-engine#9472] [Question]
My architecture is an nHost app that uses Hasura. I have extended my graphql schema with a remote schema from another service that i am running. 

In my local dev environment i would like to set the url of this remote schema to be localhost:port, and when i deploy, i want it to use the production service that i have deployed i Heroku.

Is there a way to do this, whithout changeing the values manually?

**Comment by adas98012** [comment]
@henriktorget 
You have several options to use environment specific values. Here is our [Reference Guide](https://hasura.io/docs/latest/hasura-cli/config-reference/#introduction) (please note the order of precedence).
